### ISSUE TITLE
Add ability to create short aliases for URL for cleaner links to notebooks

### ISSUE THREAD

**Issue by kyldvs** [Sketch-sh/sketch-sh#86]
Would be nice if I could set the URL for my notes

Like: https://sketch.sh/s/kyldvs/hello-world

Instead of: https://sketch.sh/s/jrBiLD8KGLXY7UFCejJHnm/BIUwNmD2AEDqkCcwBMg---N4IgRgNg9gxg1gZxALgNqgJYBMUgOIBCYAIgBwBsArgIp4BeA0pXQHYBuBbAjgKJhvkQAGhBYAhgBcxKUHAwscyEDChYApsJBsxEShqUQ1EgAQAPYwF5jAHRAAVABYYEx58YBWlBCbHGJa71sAbmsWAAcAJ3kJAH1vKJYAcwAKUwBKIJAAXyyAXSygA

**Comment by kyldvs** [comment]
Ah, it's not obvious that you can drop the last part! That solves most of this issue.

I think it could still be a nice feature, but if linking by just the note-id works, this is definitely much lower pri than other work that could be done.

I'll close this out for now.

**Comment by thangngoc89** [comment]
The intention was you should share the whole url, So I made it not obvious ;)

**Comment by kyldvs** [comment]
Fair enough, a "copy short url" or something might be a good trade off to make this discoverable, if you choose to. I think having a nice clean URL is something that can be valuable in certain situations (for example you want to link here but can't change the text displayed due to the medium you are communicating in)

**Comment by thangngoc89** [comment]
that's reasonable to me. so a share link with option to include or exclude the encoded data. with explanations for the trade-offs
